site/docs/4.1/components/alerts.md

```diff
@@ -76,7 +76,10 @@ You can see this in action with a live demo:
 Enable dismissal of an alert via JavaScript:
 
 {% highlight js %}
-$('.alert').alert()
+var alertList = document.querySelectorAll('.alert')
+alertList.forEach(function (alert) {
```

Comment:
Maybe
```js
var alertList = document.querySelectorAll('.alert')
Array.from(alertList).forEach(...);
```
?

Reply:
`Array.from` isn't widely supported and we can use `forEach` on NodeList so I won't change this line but thank you for your feedback @vsn4ik it's always appreciated 😉